R/geoserver_tools.R

```diff
@@ -39,22 +39,39 @@ query_usgs_geoserver <- function(AOI = NULL,  ids = NULL,
   if(is.na(t_srs))  { t_srs  <- 4326}
 
   source <- data.frame(server = 'wmadata',
-                       user_call  = c('huc08','huc12',
+                       user_call  = c('huc02', 'huc04', 'huc06',
+                                      'huc08', 'huc10', 'huc12',
+                                      'huc08_legacy', "huc12_nhdplusv2",
                                       'nhd','catchment', 'nhdarea',
                                       'nonnetwork',
                                       'waterbodies',
                                       'gagesII', "gagesII-basin"),
-                       geoserver  = c("huc08","huc12",
+                       geoserver  = c("wbd02_20201006", "wbd04_20201006",
+                                      "wbd06_20201006", "wbd08_20201006",
+                                      "wbd10_20201006", "wbd12_20201006",
+                                      "huc08", "huc12",
                                       "nhdflowline_network", "catchmentsp", 'nhdarea',
                                       "nhdflowline_nonnetwork",
                                       "nhdwaterbody",
                                       "gagesii", "gagesii_basins"),
-                       ids        = c("huc8", "huc12",
+                       geom_name = c("SHAPE", "SHAPE", "SHAPE",
+                                     "SHAPE","SHAPE","SHAPE",
+                                     "the_geom", "the_geom",
```

Comment:
is it really "the_geom" haha?!

Reply:
Yeah... that's a convention from ogr I think.